File: The Arcade VB/The Arcade VB/Que numero/Juego2.vb
Imports The_Arcade_VB

Public Class Juego2
    Implements IQueNumero
    Public Sub IQueNumero_Start() Implements IQueNumero.Start
        Console.Clear()
        Dim Min As Integer = 1, Max As Integer = 51
        Console.Title = "QUE NUMERO?"
        Console.WriteLine("Bienvenido a Que numero?! " & vbLf & vbLf & " Voy a pensar un numero del " & Min & " al " & (Max - 1) & " y debes adivinarlo!")
        Console.Write("Escribe un numero: ")
        Dim r As Random = New Random(Guid.NewGuid().GetHashCode())
        Dim numeroElegido As Integer = r.[Next](Min, Max)
        Dim numeroIngresado As Integer = 0

        Do
            numeroIngresado = Convert.ToInt32(Console.ReadLine())

            If numeroIngresado > numeroElegido Then
                Console.WriteLine("Es un numero menor a ese")
            ElseIf numeroIngresado < numeroElegido Then
                Console.WriteLine(" Es un numero mayor a ese")
            Else
                Console.WriteLine("Ganaste")
                Console.ReadKey()
            End If
        Loop While numeroElegido <> numeroIngresado

        Console.ReadLine()
        IQueNumero_Abrir()
    End Sub

    Private Sub IQueNumero_Abrir() Implements IQueNumero.Abrir
        Console.WriteLine("Has clic en enter para ir al menu principal")
        Console.ReadLine()
        Dim open As MenuPrincipal = New MenuPrincipal()
        open.Inicio()
    End Sub
End Class

File: The Arcade/The Arcade/Que numero/Juego2.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace The_Arcade.Que_numero
{
    class Juego2 : IQueNumero
    {
        public void Abrir()
        {
            Console.WriteLine("Has clic en enter para ir al menu principal");
            Console.ReadLine();
            MenuPrincipal open = new MenuPrincipal();
            open.Inicio();
        }

        public void Start()
        {
            Console.Clear();
            int Min = 1, Max = 51;
            Console.Title = "QUE NUMERO?";
            Console.WriteLine("Bienvenido a Que numero?! \n\n Voy a pensar un numero del " + Min + " al " + (Max - 1) + " y debes adivinarlo!");
            Console.Write("Escribe un numero: ");
            Random r = new Random(Guid.NewGuid().GetHashCode());
            int numeroElegido = r.Next(Min, Max);
            int numeroIngresado = 0;

            do
            {
                numeroIngresado = Convert.ToInt32(Console.ReadLine());

                if (numeroIngresado > numeroElegido)
                {
                    Console.WriteLine("Es un numero menor a ese");
                }
                else if (numeroIngresado < numeroElegido)
                {
                    Console.WriteLine(" Es un numero mayor a ese");
                }
                else
                {
                    Console.WriteLine("Ganaste");
                    Console.ReadKey();
                }
            } while (numeroElegido != numeroIngresado);
            Console.ReadLine();
            Abrir();
        }
    }
}
